### ISSUE TITLE
Get information of currently playing episode

### ISSUE THREAD

**Issue by kaustubhhiware** [kaustubhhiware/rose#7]
Related to #3.

Say, you're watching an episode on VLC or some video player. The work here, would be a new script, call it `rose_listen.py`. 

## Fetches information of currently playing episode

This is possible via playerctl in linux. Let's focus on VLC for the time-being, due to its simplicity.
Here's a sample output
```
> playerctl metadata # when video is playing
{'mpris:trackid': <objectpath '/org/videolan/vlc/playlist/4'>, 'xesam:url': <'file:///run/media/kaustubh/Windows/Music/FILE_NAME.mp4'>, 'vlc:time': <uint32 202>, 'mpris:length': <int64 202896000>, 'vlc:length': <int64 202896>, 'vlc:publisher': <5>}

> playerctl metadata xesam:url
file:///run/media/kaustubh/Windows/Music/FILE_NAME.mp4
```

## Getting episode information 

Assume the file name has Show title, and "S05E12" somewhere present. Assume simplest case for initial development. 

What information should be fetched? 
* Cast
* IMDB rating
* Quotes ? Information is available on IMDB page. See [this](https://www.imdb.com/title/tt1607758/#quotes). 

Quotes fetching not a priority.

**Comment by agarwalrounak** [comment]
I would like to work on the issue.

**Comment by kaustubhhiware** [comment]
Sure !
Just have a look at the way we're implementing fetching a page in the existing code, and try to have the same variables as much as possible.

Assigning this issue to you.

**Comment by kaustubhhiware** [comment]
Fixed with #18
